### ISSUE TITLE
Failure in `GetStructFromInchi()` with v1.07.2 that was previously not there with v1.06

### ISSUE THREAD

**Issue by greglandrum** [IUPAC-InChI/InChI#67]
Converting this InChI into a molecule with v1.06:
```
InChI=1S/C18H16N4O3S/c1-12-2-8-15(9-3-12)26-11-17(24)21-13-4-6-14(7-5-13)22-18(25)16(23)10-20-19/h2-10,19,23H,11H2,1H3/p+1/b16-10+
```
produces the following warning, but does succeed.
```
WARNING:  Problems/mismatches: Mobile-H( Stereobonds/cumulenes: Extra undefined)
```
with 1.07.2 (from the `rwth` branch pulled this morning), I get a return value of 2 and the error message:
```
ERROR: Add/Remove protons error; *Conversion failed*
```

The InChI in question is from this pubchem compound:
https://pubchem.ncbi.nlm.nih.gov/compound/10785244

**Comment by JanCBrammer** [comment]
Replicated over here: https://github.com/IUPAC-InChI/InChI/blob/rwth/INCHI-1-TEST/tests/test_executable/test_github_67.py.

**Comment by djb-rwth** [comment]
Hi @greglandrum,
Sorry for the delayed response as I have been sick for more than a week now.
Please be so kind as to let me know if the fixes applied in the latest release candidate in [rwth](https://github.com/IUPAC-InChI/InChI/tree/rwth) branch have resolved this issue.

**Comment by greglandrum** [comment]
@djb-rwth, yes, with the changes on the rwth branch all of the RDKit tests pass.

**Comment by greglandrum** [comment]
@djb-rwth thanks for the fix!

**Comment by djb-rwth** [comment]
> @djb-rwth thanks for the fix!

Hi @greglandrum,
Thank you for creating all these essential issues and providing quick feedbacks on their fixes.

**Comment by djb-rwth** [comment]
Closing with commit 8ecf678670f4a7f4bb5b10f00af787c06f601962.

**Comment by djb-rwth** [comment]
Hi @greglandrum,
Just wished to let you know that InChI version 1.07.2 has been released.
Hope that it will be integrated in your environment without any issues, and that InChI will work even more efficiently now.
Thank you a lot for your issues and comments, and time invested in testing the working versions as these have proven to be essential.
